planning.ipynb

```diff
@@ -342,22 +515,29 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As we can see, it hasn't completed the goal. Now, we define the sequence of actions that it should take in order to achieve\n",
-    "the goal. Then the `airCargo` acts on each of them."
+    "It will give us False because the goal state is not yet reached. Now, we define the sequence of actions that it should take in order to achieve the goal. Then the `airCargo` acts on each of them.\n",
+    "\n",
+    "Let us have an idea of the actions available to us i.e, Load, Unload, Fly\n",
+    "\n",
+    "**Load(c, p, a):** Load cargo **' c '** into plane **' p '** from airport **' a '**.\n",
```

Comment:
For this line and the two lines below, you have single quotation marks around the variable names, but you leave a space around the variable name. This can be removed, I think it will look better.